src/cdev/commands/initializer.py

```diff
@@ -131,58 +125,56 @@ def create_project(project_name: str, base_directory: DirectoryPath = None):
         }
 
         new_project.create_environment(environment, environment_settings)
-    
 
     new_project.set_state(Project_State.UNINITIALIZED)
-
     new_project.set_current_environment(DEFAULT_ENVIRONMENTS[-1])
 
 
-def _create_folder_structure(base_directory: DirectoryPath, extra_settings: List[str]):
+def _create_folder_structure(base_directory: DirectoryPath, extra_settings: List[str]) -> None:
     """Create a skeleton file structure needed to make a project.
 
     Args:
         base_directory (DirectoryPath): [description]
         extra_settings (List[str]): [description]
     """
 
-    if not os.path.isdir(os.path.join(base_directory, CDEV_FOLDER)):
+    if os.path.isdir(os.path.join(base_directory, CDEV_FOLDER)) is False:
         os.mkdir(os.path.join(base_directory, CDEV_FOLDER))
 
-    if not os.path.isdir(os.path.join(base_directory, CDEV_FOLDER, STATE_FOLDER)):
+    if os.path.isdir(os.path.join(base_directory, CDEV_FOLDER, STATE_FOLDER)) is False:
         os.mkdir(os.path.join(base_directory, CDEV_FOLDER, STATE_FOLDER))
 
-    if not os.path.isdir(os.path.join(base_directory, CDEV_FOLDER, INTERMEDIATE_FOLDER)):
+    if os.path.isdir(os.path.join(base_directory, CDEV_FOLDER, INTERMEDIATE_FOLDER)) is False:
         os.mkdir(os.path.join(base_directory, CDEV_FOLDER, INTERMEDIATE_FOLDER))
 
-
     base_settings_folder = os.path.join(base_directory, SETTINGS_FOLDER_NAME)
 
-    if not os.path.isdir(base_settings_folder):
+    if os.path.isdir(base_settings_folder) is False:
         os.mkdir(base_settings_folder)
 
-    with open(os.path.join( base_settings_folder, f'base_settings.py'), 'w'):
+    with open(os.path.join(base_settings_folder, f'base_settings.py'), 'w'):
         pass
 
-    with open(os.path.join( base_settings_folder, f'__init__.py'), 'w'):
+    with open(os.path.join(base_settings_folder, f'__init__.py'), 'w'):
         pass
     
     for environment in extra_settings:
-        with open( os.path.join( base_settings_folder, f'{environment}_settings.py'), 'w' ):
+        with open(os.path.join( base_settings_folder, f'{environment}_settings.py'), 'w'):
             pass
 
         os.mkdir(os.path.join(base_settings_folder, f'{environment}_secrets'))
 
 
-def load_project(args):
+# ANIBAL we should refactor these two into a single function with an extra parameter initialize=False
+def load_project(args) -> None:
```

Comment:
Yea that sounds like a good idea. It looks like these two are only used to wrap different Cdev commands in the proper setup flows, so we should only need to update /src/cdev/cli/__main__.py